requirements.txt

```diff
@@ -24,6 +24,8 @@ pyramid_mako
 pyramid_tm
 python-bugzilla
 python-fedora
+python-hawkey
```

Comment:
There is something called hawkey, but it has no description so I have no idea if it's the same one:

https://pypi.org/project/hawkey/

Anyways, CI is still failing due to hawkey:

```
pip	Collecting python-hawkey (from -r /bodhi/requirements.txt (line 27))
pip	[91m  Could not find a version that satisfies the requirement python-hawkey (from -r /bodhi/requirements.txt (line 27)) (from versions: )
pip	[0m[91mNo matching distribution found for python-hawkey (from -r /bodhi/requirements.txt (line 27))
```

Reply:
I'm going to just drop hawkey from requirements.txt and document it in the release notes. Then we can get this PR moving along.